docs/building-your-bot-on-iadvize.md

```diff
@@ -346,7 +346,7 @@ Here is a full conversation example:
 ### Create a conversation
 Everytime a conversation starts, this endpoint is called. It allows iAdvize to notify your bot a conversation starts.
 
-⚠️ Do not include any replies (leave the array empty) as another call to `POST /conversations/conversationId/messages` is triggered right after `POST /conversations`. It will be the right time to answer the visitor. ⚠️
+⚠️ Unless you'll implement the optional `GET /bots/:idOperator/conversation-first-messages` route, leave the `replies` array empty as another call to `POST /conversations/conversationId/messages` is triggered right after `POST /conversations`. It will be the right time to answer the visitor.
```

Comment:
Je pense qu'on peut dire  _Unless you implement..._, ça fait bizarre en future sinon.

Reply:
En effet :D